multiview_mapping_toolkit/meshes/meshes.py

```diff
@@ -1559,6 +1559,8 @@ def vis(
             vis_scalars = np.clip(vis_scalars, 0, 255).astype(np.uint8)
 
         scalar_bar_args = {"vertical": True}
+        if self.IDs_to_labels is not None:
```

Comment:
Minor: can you use `self.get_IDs_to_labels()`? Currently it just returns `self.IDs_to_labels` but in the future it might do more error checking. 

This is tangential, but it's not uncommon to compute a value and then use it if it's not null. The naive approach would be to store it as a variable before the conditional, or re-compute it inside the conditional. The first adds and extra line and the second could double the processing time if it's expensive to compute (note that in this case it's basically free). Python 3.9 introduces a silly idiom to get around this called the walrus operator, `:=`. It works like a C++ assignment operator, where you can set the value and still use it in that operation. This allows you to do the following.
```
if (maybe_null_var := expensive_operation()) is not None:
    do_something_with(maybe_null_var)
 ```
 I just wanted to mention this because I've started using it a bit. I'm on the fence whether it's useful or just confusing. But since we require 3.9+, I don't think there's harm in using it sparingly.

Reply:
Updated the code to use the method self.get_IDs_to_labels().